### PR TITLE
fix(cli): help for unknown commands

### PR DESCRIPTION
**Pull request by sanscheese**
In response to https://github.com/gridsome/gridsome/issues/1106

### CODE REVIEW

File: packages/cli/bin/gridsome.js [hunk 1]
```diff
@@ -59,8 +59,9 @@ program.arguments('<command>').action(async command => {
     const useYarn = await hasYarn()
 
     console.log()
-    console.log(`  Please run ${chalk.cyan(useYarn ? 'yarn' : 'npm install')} to install dependencies first.`)
+    console.log(chalk.red(`Unknown command ${chalk.bold(command)}`))
```
Comment (jamesgeorge007) [inline packages/cli/bin/gridsome.js:62]: Why not taking it outside the `if/else` block?

Comment (jamesgeorge007) [reply packages/cli/bin/gridsome.js:62]:
```js
program.arguments('<command>').action(async command => {
  const { isGridsomeProject, hasYarn } = require('../lib/utils')
  const availableCommands = program.commands.map(cmd => cmd._name)
  const suggestion = availableCommands.find(cmd => {
    const steps = leven(cmd, command)
    return steps < 3
  })

  console.log(chalk.red(`Unknown command ${chalk.bold(command)}`))

  if (isGridsomeProject(pkgPath) && !suggestion) {
    const useYarn = await hasYarn()

    console.log()
    console.log()
    program.outputHelp()
  } else if (suggestion) {
      console.log()
      console.log(`Did you mean ${suggestion}?`)
  }
})
```

Comment (hjvedvik) [reply packages/cli/bin/gridsome.js:62]: Yes, that seems like a better solution @jamesgeorge007 :)

Comment (sanscheese) [reply packages/cli/bin/gridsome.js]: @jamesgeorge007 Cheers for that. Agree. Btw, is the change to `suggestion` intentional?

```js
const suggestion = didYouMean(command, availableCommands)
```

```js
const suggestion = availableCommands.find(cmd => {
    const steps = leven(cmd, command)
    return steps < 3
  })
```

Comment (jamesgeorge007) [reply packages/cli/bin/gridsome.js]: #1016

Comment (jamesgeorge007) [reply packages/cli/bin/gridsome.js]: #1001

**Pull request merged**